pkg/channel/distributed/common/kafka/README.md

```diff
@@ -46,13 +47,11 @@ layer. Instead, users are required to either use their
 [REST API](https://docs.microsoft.com/en-us/rest/api/eventhub/) or the
 [go-client](https://github.com/Azure/azure-event-hubs-go/tree/master). The
 go-client is easier to use but only provides limited EventHub related
-functionality (e.g - No "namespace" management, etc...) This implementation is
+functionality (e.g. - No "namespace" management, etc...) This implementation is
 currently based on the go-client and expects Azure Namespaces to be manually
 allocated and pre-existing. The implementation does, however, abstract away
 these Azure Namespaces so that a user's Azure Subscription looks like one big
-allocation of possible EventHubs. The creation of new Topics / EventHubs will be
-load-balanced across the available Azure EventHub Namespaces as identified by
-their K8S Secret (instead of dynamic lookup via the Azure REST API).
+allocation of possible EventHubs.
 
```

Comment:
Not your fault (thanks for catching this) but... I don't think this is true any longer.  When I removed the ability to support multiple Kafka Secrets for Azure we lost the ability to span multiple Azure EventHub Namespaces and are now limited to one.   How about something like ...

```
While Azure EventHubs support the standard Kafka interfaces for Producer /
Consumer based event transmission, it does not yet provide a similar management
layer. Instead, users are required to either use their
[REST API](https://docs.microsoft.com/en-us/rest/api/eventhub/) or the
[go-client](https://github.com/Azure/azure-event-hubs-go/tree/master). The
go-client is easier to use but only provides limited EventHub related
functionality (e.g. - No "namespace" management, etc...) This implementation is
currently based on the go-client and expects Azure Namespaces to be manually
allocated and pre-existing.  Since Azure requires unique authentication for each
EventHub Namespace, we are currently limited to supporting a single instance
with its inherent limitations as to the number of Topics that can be created.
```

Reply:
Modified as suggested